azure-devops/run_build.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       displayName: 'Run vcpkg'
       timeoutInMinutes: 10
       inputs:
-        vcpkgArguments: '@$(vcpkgResponseFile)'
+        vcpkgArguments: 'boost-math:${{ parameters.targetPlatform }}-windows'

```

Comment:
Don't we need to build boost-build:x86-windows before the ARM ones?

Reply:
Isn't that taken care of by the task on line 29? (I suspect that the `vcpkg_windows.txt` listing `boost-build:x86-windows` was redundant with that.)

Reply:
Yes, I missed that.